stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -54,6 +54,9 @@ spec:
             set -ex
             cp /configmap/* /etc/rabbitmq
             rm -f /var/lib/rabbitmq/.erlang.cookie
+            {{- if .Values.forceCookie }}
```

Comment:
Let's remove `forceCookie` option and replace `rm -f /var/lib/rabbitmq/.erlang.cookie` with `echo "${RABBITMQ_ERLANG_COOKIE}" > /var/lib/rabbitmq/.erlang.cookie`

Reply:
ok, done